### PR TITLE
Guide to timestamps blog post

### PR DESCRIPTION
**Pull request by drewgeraets**
image

### CODE REVIEW

File: _posts/2023-12-20-ultimate-guide-to-timestamps/index.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  - `Oct`, `Nov`, `Dec`
+- Omit periods in abbreviations
+  - `Oct 12`,  `7:28 pm`
+- Lowercase am and pm (for 12-hour clock)
+  -  `6:38 pm`
+- Show trailing minutes at the top of the hour (to avoid any ambiguity)
+  -  `9:00 pm`
+- 12/24 hour display should determined by browser/OS/localization
+  - `1:34 pm` = `13:34`
+
```
Comment (krishanmarco) [inline _posts/2023-12-20-ultimate-guide-to-timestamps/index.md:35]: How about adding an example at the end? Eg.
```
Hard to read: 12/13/23
You understand it as soon as your eye lands on it: Dec 13, 2023
```

**Review by krishanmarco — COMMENTED** [review]
Nice one @drewgeraets

File: _posts/2023-12-20-ultimate-guide-to-timestamps/index.md (hunk 1 above)
Comment (drewgeraets) [reply _posts/2023-12-20-ultimate-guide-to-timestamps/index.md:35]: Great suggestion. I'll add.

**Comment by drewgeraets** [comment]
@krishanmarco I made some changes to the blog, itself, if you wouldn't mind taking a quick look again.

**Comment by SuleimanAbdulshaqur** [comment]
LGTM

**Review by SuleimanAbdulshaqur — APPROVED** [review]
LGTM

**Pull request merged**